### DISCUSSION TITLE
surveymarks/et-5-crh/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#25] [Announcements]
# surveymarks/et-5-crh/

http://localhost:4000/surveymarks/et-5-crh/

**Comment by jengalas** [comment]
Posted by Eric Czerwinski
June 21, 2023 at 10:06 am
_Imported from WordPress_

This is very strange and perhaps I am not completely understanding the concept of an Electronic Traverse Station. There exists another USGS ET 5 CRH 1964 located on the Appalachian Trail about 0.4mi from the top of the North Kinsman Peak in New Hampshire. 44N 8’17” 71W 44′ 3″
